lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -513,6 +516,7 @@ func (handler *ContainersHandlersImpl) RenameContainerHandler(params containers.
 	}
 
 	h = h.Rename(op, params.Name)
+	handler.netCtx.UpdateContainerNameInScope(h)
```

Comment:
Are there any other cases to call this `UpdateContainerNameInScope` function besides `h.Rename`?

Reply:
No, this is a new method for rename ops.